continuous_integration/recipe/conda_build_config.yaml

```diff
@@ -4,5 +4,5 @@ rust_compiler_version:
     - 1.69
 libprotobuf:
     - 3
-setuptools_rust:
-    - 1.5.2
+maturin:
+    - 0.15.3
```

Comment:
Could drop the patch release to simply get the latest of `0.15`

```suggestion
    - 0.15
```

Reply:
Yup that makes sense, going to try bumping us to a more recent maturin version but if that doesn't work will go with this suggestion